python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/router_bgp.schema.yml

```diff
@@ -1484,6 +1486,26 @@ keys:
                       - str
                     min: 2
                     max: 64
+          evpn_ethernet_segment:
+            type: list
+            primary_key: domain
+            items:
+              type: dict
+              keys:
+                domain:
```

Comment:
`domain` should be a `required` key

Reply:
It is already since it is the `primary_key`. No need to also set it in the schema.